arthur/apis/cloudflare/zones.py

```diff
@@ -0,0 +1,37 @@
+"""APIs for managing Cloudflare zones."""
+from typing import Optional
+
+import aiohttp
+
+from arthur.config import CONFIG
+
+CF_TOKEN = CONFIG.cloudflare_token
+
+
+async def list_zones(zone_name: Optional[str] = None) -> dict[str, str]:
+    """List all Cloudflare zones."""
+    endpoint = "https://api.cloudflare.com/client/v4/zones"
+    request_headers = {"X-Auth-User-Service-Key": CF_TOKEN}
```

Comment:
Is this actually the way of authorizing? Isn't it through the Authorization header?

Look at how things are done here:

https://github.com/jakejarvis/cloudflare-purge-action/blob/eee6dba0236093358f25bb1581bd615dc8b3d8e3/entrypoint.sh#L55-L61

On top of that, we should standardise the headers for the methods in a constant at the top of the file.